internal/provider/data_source_policy_set_test.go

```diff
@@ -248,6 +258,11 @@ resource "tfe_project_policy_set" "foobar" {
 	project_id = tfe_project.foobar.id
 }
 
+resource "tfe_workspace_policy_set_exclusion" "foobar" {
```

Comment:
Awesome work! Reviewed this PR and realized a new resource is being used in this test, so let's hold off on merging so I can review the PR that adds the resource you're using here.